### PR TITLE
Apache installation and configuration doc update

### LINKED ISSUE #1123: Installation.rst points to develop branch
Needs to be updated to point at master branch.

> curl https://raw.githubusercontent.com/ga4gh/server/develop/scripts/demo_example.py

### PR DESCRIPTION
**Pull request by kozbo**
This further addresses #821 and should close #1123 
Also added a section discussing the process and thread count settings for Apache.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -101,7 +103,33 @@ and insert the following contents towards the end of the file
         Require all granted
     </Directory>
 
-Restart Apache:
+Note: be sure to keep the number of threads limited to 1 in the WWSGIDaemonProcess
```
Comment (jeromekelleher) [inline docs/installation.rst]: I think we could put this in a ``..warning`` block to really emphasise it --- this is a nasty gotcha!
Also, extra 'W' in WWSGIDaemonProcess

File: docs/installation.rst [hunk 2]
```diff
@@ -139,16 +167,18 @@ how to deploy on various other servers.
...
 +++++++++++++++
 
-If you are encountering difficulties getting the above to work, it is helpful
-to turn on debugging output. Do this by adding the following line to your
-config file:
+Server errors will be output to the web server's error log by default (in Apache on
+Debian/Ubuntu, for example, this is ``/var/log/apache2/error.log``). Each client
+request will be logged to the web server's access log (in Apache on Debian/Ubuntu
+this is ``/var/log/apache2/access.log``). If you want more debug output then you
+can try activating flask debug mode by adding the following line to your config
```
Comment (jeromekelleher) [inline docs/installation.rst]: I'm not sure we should be recommending admins turn on Flask's DEBUG mode any more --- does it give anything extra?

**Comment by kozbo** [comment]
@jeromekelleher warning added. Might the flask debug mode be useful to developers who want to use the interactive debugger?

**Comment by jeromekelleher** [comment]
Thanks @kozbo. I think we should keep development and deployment considerations separate where possible. I think it just causes potential confusion to mention DEBUG here at all in the deployment section. Developers will almost always be using the development server CLI in any case, I would imagine.

**Comment by kozbo** [comment]
Debug setting description removed. Ready for merge?

**Comment by jeromekelleher** [comment]
Great, thanks @kozbo. +1.

Could you squash these commits please? For small changes like this it's good to keep it to a single commit --- it makes life a bit easier when doing large merges between branches.

**Comment by kozbo** [comment]
OK, should be good to go.

**Pull request merged**